bundle/manifests/kataconfiguration.openshift.io_kataconfigs.yaml

```diff
@@ -81,6 +87,8 @@ spec:
                       are ANDed.
                     type: object
                 type: object
+            required:
```

Comment:
> @bpradipt I believe that the way you approached this is the correct one, but this requires a big warning in the release note, since IIUC (at least when `checkNodeReadiness` is set) you would now need to label nodes explicitly to be able to deploy, which you did not need to do before.

@c3d right. Doc changes are queued next for this.